### ISSUE TITLE
Update peer dependencies in the NPM file

### ISSUE THREAD

**Issue by Gimly** [swimlane/ngx-charts-dag#28]
It would be great if we could update the peer dependencies in the NPM file to something more recent. Maybe using a caret ^ instead of a tilde would also make the upgrade simpler.

```
"@angular/common": "~4.0.1",
 "@angular/core": "~4.0.1",
```
should be
```
"@angular/common": "^4.2.4",
 "@angular/core": "^4.2.4",
```

**Comment by justindiaw** [comment]
Same issue here. Seems like it won't meet higher angular dependencies.

**Comment by marjan-georgiev** [comment]
This is resolved in 3.0.0
